### DISCUSSION TITLE
Is the dictionary available in dic format?

### DISCUSSION THREAD

**Discussion by mrsanor** [voikko/corevoikko#58] [Q&A] [answered]
Hello, I was wondering if the dictionary was available in Hunspell dic format? Like the dictionaries in here: https://github.com/wooorm/dictionaries/tree/main

Or if you have any tips on how to transform this dictionary into that format?

**Comment by ahomansikka** [comment, marked as answer]
No Hunspell dic format.

Tips. After a few minutes of thinking perhaps  you must write a translator that translates the language Voikko uses (foma) to the language Hunspell uses. That may be like writing a program that translates Ada to Java...

**Reply by mrsanor** [reply]
Okay, thank you. I will have to look into this.
